homeassistant/components/evohome/climate.py

```diff
@@ -362,15 +409,19 @@ def current_temperature(self) -> float | None:
         Controllers do not have a current temp, but one is expected by HA.
         """
         temps = [
-            z.temperatureStatus["temperature"]
+            z.temperatureStatus["temperature"]  # type: ignore[index,unused-ignore]
             for z in self._evo_tcs.zones.values()
-            if z.temperatureStatus["isAvailable"]
+            if z.temperatureStatus.get("isAvailable")  # type: ignore[union-attr,unused-ignore]
```

Comment:
Note use of `unused-ignore`. The original error was:
`Item "None" of "dict[str, Any] | None" has no attribute "get"  [union-attr]`

Reply:
Same as above.